scripts/fetcher.js

```diff
@@ -91,15 +87,17 @@ Fetcher.prototype.loadManifest = function(url){
     .then(res => {
       if (!res.ok){
         Log.error("DL", "could not fetch manifest");
-        ErrorPopUp("Can not download the manifest file");
+        $("#modalMessage").html("Can not download the manifest file");
+        $("#warningPopup").modal();
         throw "DLerror";
       }
       return res.text();
     })
     .then(str => (new window.DOMParser()).parseFromString(str, "text/xml"))
     .then(data => this.onManifestLoaded(data))
     .catch(err => { 
-      ErrorPopUp("Not available url <br> url: " + url);
```

Comment:
@JWSon I remember I put all the modal stuff to ErrorPopUp function (so we can re-use it everywhere) why are we switching back in this commit?